python_modules/dagster/dagster/_core/storage/runs/sql_run_storage.py

```diff
@@ -791,6 +797,11 @@ def has_bulk_actions_selector_cols(self) -> bool:
             ]
             return "selector_id" in column_names
 
+    def has_backfill_id_column(self) -> bool:
```

Comment:
i didn't see any examples where the we determined if the data migration was done separately from determining if the columns existed. the ones i found used column existence to assert that the data migration was done too. So i'm just following that same pattern here

Reply:
this is done for partitions. see `self.has_built_index`.